Index: OpenTRACK.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>f = open(\"track.csv\", \"r\")\n\nprint(f.readlines())
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/OpenTRACK.py b/OpenTRACK.py
--- a/OpenTRACK.py	(revision 78e9a71a1541ff919608fbea2a60668751521db6)
+++ b/OpenTRACK.py	(date 1725611966258)
@@ -1,3 +1,1 @@
-f = open("track.csv", "r")
-
-print(f.readlines())
\ No newline at end of file
+f = open("track.csv", "r")
\ No newline at end of file
